helm/resource_repository_test.go

```diff
@@ -52,19 +52,28 @@ func testAccHelmRepositoryConfigBasic(name, url string) string {
 	`, name, url)
 }
 
-func testAccCheckHelmRepositoryDestroy(s *terraform.State) error {
+func testAccPreCheckHelmRepositoryDestroy(t *testing.T, name string) {
```

Comment:
Since this test file may disappear in the future as repository resource is deprecated, I guess this function should be moved somewhere else, either `data_repository_test.go` or `resource_release_test.go`. What do you think?

Reply:
Indeed, moved it to `resource_release_test.go` where it is used.